grpcdogfood/Makefile

```diff
@@ -0,0 +1,52 @@
+.PHONY: dogfood-go-client dogfood-go-server
```

Comment:
I am not sure what `.PHONY` does

Reply:
I think [this answer](https://stackoverflow.com/a/3574387) explains it better than I would do with my own words :)